src/api/services/actions/issuancesActions.jsx

```diff
@@ -6,7 +6,7 @@ export const fetchIssuances = async (page = 0, size = 10, searchTerm = "") => {
     const response = await get(`${ISSUANCE_BASE_URL}/list`, {
       page,
       size,
-      search: encodeURIComponent(searchTerm),
+      search: searchTerm,
     });
 
     return response;
```

Comment:
don't directly return the response. send it in key value parin for get requests as well.